compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.cpp

```diff
@@ -29,10 +29,17 @@ static LogicalResult setRootConfigForPadPipeline(func::FuncOp entryPointFn,
       IREE::Codegen::DispatchLoweringPassPipeline::None);
 }
 
-static SmallVector<int64_t> getPackedSize(Operation *op) {
+static SmallVector<int64_t> getPackedSize(Operation *op, const int packLevel) {
```

Comment:
Maybe `packLevel` can be made an `enum`?

Reply:
Here I'd like to use `int` to keep consistent with the other parts we used for packLevel, padLevel and bufferizeLevel.